chapter_recurrent-modern/lstm_vn.md

```diff
@@ -96,14 +101,14 @@ where $\mathbf{W}_{xi}, \mathbf{W}_{xf}, \mathbf{W}_{xo} \in \mathbb{R}^{d \time
 are weight parameters and $\mathbf{b}_i, \mathbf{b}_f, \mathbf{b}_o \in \mathbb{R}^{1 \times h}$ are bias parameters.
 -->
 
-*dịch đoạn phía trên*
+trong đó $\mathbf{W}_{xi}, \mathbf{W}_{xf}, \mathbf{W}_{xo} \in \mathbb{R}^{d \times h}$ và $\mathbf{W}_{hi}, \mathbf{W}_{hf}, \mathbf{W}_{ho} \in \mathbb{R}^{h \times h}$ là các trọng số và $\mathbf{b}_i, \mathbf{b}_f, \mathbf{b}_o \in \mathbb{R}^{1 \times h}$ là các hệ số điều chỉnh.
 
 
 <!--
 ### Candidate Memory Cell
 -->
 
-### *dịch tiêu đề phía trên*
+### Ô Ký ức Tiềm năng
```

Comment:
Sorry @cuongvng mình hơi bị lộn xộn quá, mà giờ đọc lại thấy `Ô nhớ Tiềm năng` đúng hơn chứ nhỉ.
Thà như `Candidate Memory` không thì dịch `Ký ức Tiềm năng` mới hợp.

Reply:
hm chỗ này hơi nhập nhằng chút, em thấy ở dưới có nói kí hiệu của `candidate memory cell` là $\tilde{\mathbf{C}}_t $
> Since we have not specified the action of the various gates yet, we first introduce the *candidate* memory cell $\tilde{\mathbf{C}}_t $
 
trong hình thì kí hiệu đó lại là của `candidate memory` (không có `cell`) và đang dịch là `ký ức tiềm năng`
![image](https://user-images.githubusercontent.com/30251584/83655877-16667d80-a5e9-11ea-9f5f-6fe9c2ed78e3.png)

Reply:
Yeah, chỗ này mình thấy nhập nhằng là do tiếng việt nó chi tiết hơn tiếng anh: 
* Cái mà **được** ghi nhớ thì là `Kí ức` (ở đây là các **giá trị** trong C và C_hat).
* Cái mà **để** ghi nhờ thì là `Trí nhớ`, `bộ nhớ`, `ô nhớ`, `sổ ghi nhớ`, ... (ở đây là chỉ vector C, kiểu như placeholder).

Vậy nên mình thấy túm lại nếu có chữ `cell`, tức chỉ thiết bị để nhớ thì dịch là `ô nhớ (tiềm năng)`.
Đứng không như trong hình thì dịch là `Kí ức (tiềm năng)`.
Tuy không cùng là một từ nhưng nó lại phù hợp với cách dùng thông dụng trong tiếng việt. @cuongvng thấy sao?